python/torch_mlir/extras/onnx_importer.py

```diff
@@ -464,13 +464,23 @@ def type_proto_to_type(self, tp: onnx.TypeProto) -> IrType:
             # See TypeProto: sequence_type, map_type, optional_type, sparse_tensor_type.
             raise OnnxImportError(f"Unsupported ONNX TypeProto: {tp}")
 
+    def _sanitize_name(self, name):
+        try:
+            name = f"__{int(name)}__"
```

Comment:
I generally avoid exceptions in the critical/common path if feasible. How about:

```
if not name.isidentifier():
  name = "_" + name
return re.sub("[:/]", "_")
```

Reply:
Done